src/test/java/io/confluent/idesidecar/restapi/testutil/FakeSideCarProfile.java

```diff
@@ -0,0 +1,21 @@
+package io.confluent.idesidecar.restapi.testutil;
+
+import io.quarkus.test.junit.QuarkusTestProfile;
+import java.util.Map;
+
+public class FakeSideCarProfile implements QuarkusTestProfile {
+
+  /**
+   * Disable the access token filter to protest unit tests.
+   */
+  @Override
+  public Map<String, String> getConfigOverrides() {
+    return Map.of(
+        "vscode.version", "0.4.0 override",
+        "os.name", "linux-override",
```

Comment:
Is this representative of real `os.name` values? IIRC, most are space-separated with capitalized words.